pkg/controller/virtualmachinevolumeexport/virtualmachinevolumeexport_controller.go

```diff
@@ -151,8 +151,10 @@ func (r *ReconcileVirtualMachineVolumeExport) validateVirtualMachineVolume() err
 		return goerrors.New("fail to get virtual machine volume")
 	}
 	// check if vmv is available
-	if vmVolume.Status.State != hc.VirtualMachineVolumeStateAvailable {
-		return goerrors.New("virtual machine volume is not available")
+	found, cond := util.GetConditionByType(vmVolume.Status.Conditions, hc.ConditionReadyToUse)
+	if !found || cond.Status != corev1.ConditionTrue {
+		klog.Info("VirtualMachineVolume state is not available")
```

Comment:
I noticed that vmv code also has it too, but probably this log is not necessary.

Reply:
Thanks for your comment!